node/pkg/processor/cleanup.go

```diff
@@ -81,7 +81,7 @@ func (p *Processor) handleCleanup(ctx context.Context) {
 			// This occurs when we observed a message after the cluster has already reached
 			// consensus on it, causing us to never achieve quorum.
 			if ourVaa, ok := s.ourObservation.(*VAA); ok {
-				if _, err := p.getSignedVAA(*db.VaaIDFromVAA(&ourVaa.VAA)); err == nil {
+				if p.haveSignedVAA(*db.VaaIDFromVAA(&ourVaa.VAA)) {
```

Comment:
Check for err?

Reply:
If there is an error, p.haveSignedVAA() will just return false but the error is logged in p.haveSignedVAA now. What do you think?